conversion/tests/test_file_conversion.py

```diff
@@ -37,8 +34,11 @@ def test_nrrd2nifti(self):
         testing.assert_array_equal(reference_nifti_data, converted_nifti_data)
         testing.assert_array_equal(reference_nifti_affine, converted_nifti_affine)
         testing.assert_array_equal(reference_nifti_bvals, converted_nifti_bvals)
-        testing.assert_array_equal(reference_nifti_bvecs, converted_nifti_bvecs)
-
+        for bvec_idx in range(0,len(reference_nifti_bvecs)):
+            reference_bvec = array(reference_nifti_bvecs[bvec_idx])
+            converted_bvec = array(converted_nifti_bvecs[bvec_idx])
+            testing.assert_array_almost_equal_nulp(reference_bvec,
+                                                   converted_bvec, 1)
```

Comment:
This is good, you are comparing line by line as opposed to all the lines I compared together before.

Reply:
I am doing line by line. I could do all the lines together, but then I need to flatten a list of list into a single list.

Reply:
```python
#psuedocode
referenence_bvec = []
for bvec_idx  in range(0,len(reference_nifiti_bvecs):
    reference_bvec.extend(  array(reference_nifti_bvecs[bvec_idx]) )
```